app/src/main/java/ar/com/wolox/android/example/ui/root/RootPresenter.kt

```diff
@@ -0,0 +1,88 @@
+package ar.com.wolox.android.example.ui.root
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import ar.com.wolox.android.example.model.User
+import ar.com.wolox.android.example.network.UserService
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices
+import ar.com.wolox.wolmo.networking.retrofit.callback.NetworkCallback
+import okhttp3.ResponseBody
+import javax.inject.Inject
+
+class RootPresenter @Inject constructor(
+    private val mRetrofitServices: RetrofitServices
+) : BasePresenter<IRootView>() {
+
+    private var mPrefName: String? = null
+    private var mPrefEmail: String? = null
+    private var mPrefPass: String? = null
+    private var mPrefUserId: String? = null
+
+    internal lateinit var sharedPref: SharedPreferences
+
+    /**
+     * connectUser()
+     */
+    fun connectUser() {
+        val call = mRetrofitServices.getService(UserService::class.java).getUserLogin(mPrefEmail)
+
+        view.showLoading()
+
+        call.enqueue(object : NetworkCallback<List<User>>() {
+
+            override fun onResponseSuccessful(response: List<User>?) {
+
```

Comment:
Remove  this empty line. Moreover, you can use `view.apply{}` and send everyithing there